app/models/passwordless/session.rb

```diff
@@ -33,14 +31,18 @@ def timed_out?
     end
 
     def claim!
-      raise TokenAlreadyClaimedError if claimed?
+      raise Passwordless::Errors::TokenAlreadyClaimedError if claimed?
       touch(:claimed_at)
     end
 
     def claimed?
       !!claimed_at
     end
 
+    def valid_session?
```

Comment:
`session.valid_session?` has a double _session_. Let's rename to `session.valid?` ?